doc/dev/mgmt/tests.md

```diff
@@ -84,109 +69,96 @@ credentials = UserPassCredentials(
     'user@domain.com',     # Your new user
     'my_smart_password',   # Your password
 )
-```
+``` -->
 
 ### Get a token with Active Directory application and service principal
 
 Follow this detailed tutorial to set up an Active Directory application and service principal:
 https://azure.microsoft.com/documentation/articles/resource-group-create-service-principal-portal/
 
-To use the credentials from Python,
-you need the application ID (a.k.a. client ID),
-authentication key (a.k.a. client secret),
-tenant ID and subscription ID from the Azure portal for use in the next step.
-[This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-describes where to find them
-(besides the subscription ID,
-which is in the "Overview" section of the "Subscriptions" blade.)
+To use the credentials from Python, you need the application ID (a.k.a. client ID), authentication key (a.k.a. client secret), tenant ID and subscription ID from the Azure portal for use in the next step. [This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key) describes where to find them (besides the subscription ID, which is in the "Overview" section of the "Subscriptions" blade.)
+
+The recommended practice is to store these three values in environment variables called `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET`. To set an environment variable use the following commands:
+```Shell
+$env:AZURE_TENANT_ID=<value>        # PowerShell only
+set AZURE_TENANT_ID=<value>         # Windows CMD (alternatively, use export AZURE_TENANT_ID=<value> to export to the global env)
+export AZURE_TENANT_ID=<value>      # Linux shell only
+```
+*** Note: when setting these variables, do not wrap the value in quotation marks ***
 
 You are now able to log in from Python using OAuth.
 You can test with this code:
 ```python
+import os
 from azure.common.credentials import ServicePrincipalCredentials
 
 credentials = ServicePrincipalCredentials(
-    client_id =  'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL',
-    secret = 'XXXXXXXXXXXXXXXXXXXXXXXX',
-    tenant = 'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL'
+    client_id = os.environ['AZURE_CLIENT_ID'],
+    secret = os.environ['AZURE_CLIENT_SECRET'],
+    tenant = os.environ['AZURE_TENANT_ID']
 )
 ```
 
 ## Providing credentials to the tests
 
-When you run tests in playback mode,
-they use a fake credentials file,
-located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`,
-to simulate authenticating with Azure.
+When you run tests in playback mode, they use a fake credentials file, located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`, to simulate authenticating with Azure.
```

Comment:
It's probably worth noting that typically one doesn't have to adjust this.  (if you're feeling enthusiastic, can provide a token example of the rare reasons one might need to touch it).